config/plugins.js

```diff
@@ -25,7 +25,10 @@ module.exports = {
 		'import/core-modules': [
 			'electron',
 			'atom'
-		]
+		],
+		'import/parsers': {
```

Comment:
I think that should be set dynamically, only when linting a `ts` or `tsx` file. Otherwise `typescript-eslint-parser` might become needed even the current file is a regular JS file.
So in `index.js`:
```js
		if (paths.filter(filePath => isTSFile(path.extname(filePath))).length > 0) {
			opts.parser = 'typescript-eslint-parser';
            opts['import/parsers']['typescript-eslint-parser'] = ['.ts', '.tsx'];
		}
```